girder_annotation/girder_large_image_annotation/web_client/views/hierarchyWidget.js

```diff
@@ -0,0 +1,150 @@
+import $ from 'jquery';
```

Comment:
You need to add the HierarchyWidget to the views/index.js file to get it imported.

Reply:
See d350560